jsonb/src/main/java/com/inrupt/client/jsonb/JsonbService.java

```diff
@@ -39,21 +38,21 @@
  */
 public class JsonbService implements JsonService {
 
-    private final Jsonb jsonb;
+    private final JsonbConfig config;
 
     /**
      * Create a JSON-B service.
      */
     public JsonbService() {
         final JsonbConfig config = new JsonbConfig();
         config.withPropertyNamingStrategy(PropertyNamingStrategy.LOWER_CASE_WITH_DASHES);
-        this.jsonb = JsonbBuilder.create(config);
+        this.config = config;
     }
 
     @Override
     public <T> void toJson(final T object, final OutputStream output) throws IOException {
         try {
-            jsonb.toJson(object, output);
+            JsonbBuilder.create(config).toJson(object, output);
```

Comment:
Creating a Jsonb object is relatively expensive. Creating it once was intentional

Reply:
It appeared to resolve the test issues locally, but that's actually not the case depending on how I run the tests, so I reverted this and I am still investigating.